### ISSUE TITLE
Enable Discussions

### ISSUE THREAD

**Issue by data-man** [goldendict/goldendict#1345]
GitHub Discussions [now available](https://github.blog/changelog/2020-12-08-github-discussions-now-available-as-a-public-beta/) for beta testing.

It would be nice to enable them for this project.

**Comment by shinecurve** [comment]
Yes. Moreover, the compilation instructions under Windows are too brief and not detailed enough for general users.
